.env.sample

```diff
@@ -7,4 +7,5 @@ SERVER_MAIL= email address
 google_client=string from google
```

Comment:
This is supposed to be capitalised on another story which is delayed.
Kindly make the changes on this branch so we move on.

Reply:
ok